src/content/contributing/code-of-conduct/_index.en.md

```diff
@@ -4,3 +4,11 @@ date: 2020-02-19T21:43:46+01:00
 weight: 20
 ---
 
+## Submariner Community Code of Conduct
+
+Submariner follows the [CNCF Code of Conduct][1].
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the Submariner Committers.
+
+[1]: https://github.com/cncf/foundation/blob/master/code-of-conduct.md
```

Comment:
I suggest linking specifically to v1.0, so we don’t silently approve potential changes to the CNCF CoC:

```suggestion
[1]: https://github.com/cncf/foundation/blob/bec34a2614c980f8cfe38b18105e0baa820936cc/code-of-conduct.md
```

Reply:
:+1: